README.md

```diff
@@ -31,6 +31,13 @@ run "bundle install" to install dependencies
 
 - Run the command `ruby app.rb`.
 
+### Run the tests 
+
+- install spec gem by typing : `gem install rspec`
+- Go to the `__test__` directory by typing  `cd __test__`
+- run all the test by `rspec spec .`
```

Comment:
- [ ] **REQUIRED :** It will be nice to make sure the tests works. Please provide screenshots if you feel you need to.

![image](https://user-images.githubusercontent.com/2903437/165856769-b64793ea-93bb-47e5-bf13-3f7bab743715.png)